test/functional/apps/dashboard/_embeddable_rendering.js

```diff
@@ -36,8 +36,6 @@ export default function ({ getService, getPageObjects }) {
       const panelCount = await PageObjects.dashboard.getPanelCount();
       expect(panelCount).to.be(27);
 
-      await PageObjects.dashboard.waitForRenderComplete();
```

Comment:
I'm a little worried why leaving this in broke the tests.  Reporting relies on render complete so if something isn't working right, it might be a bug.  Was it just flaky or consistently broken?

Although it's in a weird spot anyway and kinda looks like it was accidentally left in (I'm sure my bad).  So I'm fine with removing it, just still curious how it caused things to fail.

Reply:
It would run locally fine with that line in. It always failed on CI if that line is in. One panel never set render to true and I could not figure out why - and why it would work locally